cmd/flux/events.go

```diff
@@ -84,14 +87,19 @@ func init() {
 		"indicate if the events should be streamed")
 	eventsCmd.Flags().StringVar(&eventArgs.forSelector, "for", "",
 		"get events for a particular object")
-	eventsCmd.Flags().StringSliceVar(&eventArgs.filterTypes, "types", []string{}, "filter events for certain types")
+	eventsCmd.Flags().StringSliceVar(&eventArgs.filterTypes, "types", []string{}, "filter events for certain types (valid types are: Normal, Warning)")
 	rootCmd.AddCommand(eventsCmd)
 }
 
 func eventsCmdRun(cmd *cobra.Command, args []string) error {
 	ctx, cancel := context.WithTimeout(context.Background(), rootArgs.timeout)
 	defer cancel()
 
+	err := validateEventTypes(eventArgs.filterTypes)
+	if err != nil {
```

Comment:
```suggestion
if 	err := validateEventTypes(eventArgs.filterTypes); err != nil {
```
This is purely my personal preference to keep `err` scoped to as little context as possible.